sll-insertion.cpp

```diff
@@ -0,0 +1,95 @@
+#include <iostream>
+using namespace std;
+
+struct Node {
+  int data;
+  struct Node *next;
+} *head = NULL;
+
+void insertAtBegin(int value) {
+  Node *newNode = new Node();
+  newNode->data = value;
+  if (head == NULL) {
+    newNode->next = NULL;
+    head = newNode;
+  }
+
+  else {
+    newNode->next = head;
+    head = newNode;
+  }
+  printf("\nOne node inserted!!!\n");
+}
+
+void insertAtEnd(int value) {
+  Node *newNode = new Node();
+  newNode->data = value;
+  newNode->next = NULL;
+  if (head == NULL)
+    head = newNode;
+  else {
+    struct Node *temp = head;
+    while (temp->next != NULL)
+      temp = temp->next;
+    temp->next = newNode;
+  }
+  printf("\nOne node inserted!!!\n");
+}
+
+void insertBetween(int value, int loc)
+
```

Comment:
Here and other places too. 
Opening brackets must be in the same line as function name.